src/partials/60_section-gallery.html

```diff
@@ -0,0 +1,329 @@
+<div class="section--gallery">
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-6">
+        <div class="panel-bar">
+          <div class="row no-gutters align-items-end">
+            <div class="col-auto heading"><h3>Furniture gallery</h3></div>
+            <div class="col"></div>
+          </div>
+        </div>
+        <div class="gallery-box">
+          <nav>
+            <div class="nav nav-tabs" id="nav-tab" role="tablist">
+              <a
+                class="nav-item nav-link bla"
+                id="nav-feature-tab"
+                data-toggle="tab"
+                href="#nav-feature"
+                role="tab"
+                aria-controls="nav-feature"
+                aria-selected="false"
+                >Feature</a
+              >
+              <a
+                class="nav-item nav-link active"
+                id="nav-top-seller-tab"
+                data-toggle="tab"
+                href="#nav-top-seller"
+                role="tab"
+                aria-controls="nav-top-seller"
+                aria-selected="true"
+                >Top seller</a
+              >
+              <a
+                class="nav-item nav-link"
+                id="nav-sale-off-tab"
+                data-toggle="tab"
+                href="#nav-sale-off"
+                role="tab"
+                aria-controls="nav-sale-off"
+                aria-selected="false"
+                >Sale off</a
+              >
+              <a
+                class="nav-item nav-link"
+                id="nav-top-rated-tab"
+                data-toggle="tab"
+                href="#nav-top-rated"
+                role="tab"
+                aria-controls="nav-top-rated"
+                aria-selected="false"
+                >Top rated</a
+              >
+            </div>
+          </nav>
+          <div class="tab-content" id="nav-tabContent">
+            <div
+              class="tab-pane fade feature"
+              id="nav-feature"
+              role="tabpanel"
+              aria-labelledby="nav-feature-tab"
+              style="background-image: url(images/gallery/chair-1.png);"
+            >
+              <div class="outlines">
+                <a
+                  href="#"
+                  class="btn-outline"
+                  data-toggle="tooltip"
+                  data-placement="right"
+                  title="Add To Wishlist"
+                  ><i class="far fa-heart"></i
+                ></a>
+                <a
+                  href="#"
+                  class="btn-outline"
+                  data-toggle="tooltip"
+                  data-placement="right"
+                  title="Add To Compare"
+                  ><i class="fas fa-exchange-alt"></i
+                ></a>
```

Comment:
zakończenie tego znacznika powinno znajdować się linię wyżej, tak samo zakończenie `a` w linii `79` i we wszystkich następnych linkach dokładnie to samo

Reply:
Taki format zapisu pojawił się automatycznie. Poprawiłam w całym pliku te znaczniki, ale niestety widzę, że przy zakomitowaniu wszystko wróciło. Tak zresztą to wygląda we wszystkich plikach projektu.